src/help/faq/README.md

```diff
@@ -47,25 +49,49 @@ In the app, go to Settings > Content Settings.
 
 ---
 
-### I am on the public version of Paperback! When will other manga sources be added?
-Currently other manga sources are already available on the patreon version of the application. Sources on the public version of the app are expected to come most likely at the end of July.
+### How do I enable Hentai
+ * Sign into [MangaDex website](https://mangadex.org/) on a browser and go to your settings to set Hentai toggle to "**View All**".
+ * Then sign into MangaDex on Paperback, hitting "Remember me", and go into your *Content Settings* in the app settings tab and choose "**Show All**".
+
+---
+
+### My content settings are reset everytime I open the app
+When you are logging in, make sure that **"Remember me"** is checked on the page.
+Also make sure that your email address is verified on MangaDex.
+
+---
+
+### How can I add other manga sources?
+Other manga sources are available on the latest TestFlight public and patreon versions of the application.
+Read [Adding an External Repository](/help/guides/adding-repos/) guide to learn how to install other sources.
 
 ---
 
 ### I am on the public version of Paperback! When can I download manga to my device for offline viewing?
-Currently downloading manga is already available on the patreon version of the application. Sources on the public version of the app are expected to come most likely at the end of July.
+Downloading manga is available on TestFlight public and patreon versions of the application. Update to the latest TestFlight build to be able to download manga.
 
 ---
 
-### How do I enable Hentai
-Go to [MangaDex Settings](https://mangadex.org/settings) and login. Look for the **Hentai toggle** and choose **Show toggle (in navbar cog)**.
-Then restart the app and go to Settings > Content Settings.
+### How can I make a backup of my library?
+**Creating a backup**
+ * Go to settings > backup
+ * Press "**Create a backup**" and choose where you want to save the backup (Note is recommended)
+
+**Restoring a backup**
+ * Reinstall all sources used at the time of the backup
```

Comment:
```suggestion

```
I’ve updated the app so this step is automatic.

Reply:
Note that this content might not have made it to the public version yet, so probably only applies to the patreon build